tools/Cron_jobs/quarantine_report.php

```diff
@@ -51,7 +51,7 @@
 $required_constant_missing_count = 0;
 foreach ($required_constant as $constant) {
     if (!defined($constant)) {
-        echo "The variable $constant is empty, please set a value in conf.php.\n";
+        echo __('message61') . "\n";
```

Comment:
line 54 should be `echo sprintf(__('message61'), $constant) . "\n";` and `message61` in translations array should be `The variable %s is empty, please set a value in conf.php.`.

`%s` is replaced by the content of `$constant`